### ISSUE TITLE
Server list bar

### ISSUE THREAD

**Issue by CamaroKing59** [Eight-P/BD.8P#23]
### T1 Issue report:

Started discord and the server list bar was bugged out.. pushed to the top of the discord and couldnt view any discords at all

### T1 version:

v0.8.1

### Discord client:

Stable (Default)

### Platform:

Windows

### Discord Mod:

BetterDiscord

**Comment by Eight-P** [comment]
Yeah, they pushed a big class change for all clients that broke the Theme.
It was re-rolled on Stable, so that should still work for the moment.
If they stick with the changes, it will take me a bit to fix it.
